rbac/internal/views.py

```diff
@@ -1325,3 +1365,32 @@ def retrieve_ungrouped_workspace(request):
         return HttpResponse(json.dumps(data), content_type="application/json", status=201)
     except Exception as e:
         return HttpResponse(str(e), status=500)
+
+
+def lookup_resource(request):
```

Comment:
maybe it will be good to put some into try catch block and proper error message.